config.js

```diff
@@ -1,27 +1,25 @@
-"use strict";
```

Comment:
I don't remember why I had `"use strict"` everywhere but should be safe to remove now